app/src/org/storymaker/app/HomeActivity.java

```diff
@@ -87,6 +87,15 @@
     public void onCreate(Bundle savedInstanceState) {
     	super.onCreate(savedInstanceState);
 
+        // set title bar as a reminder if test server is specified
```

Comment:
HomeActivity is the activity that shows up in the manifest as being labeled "StoryMaker 2".  Did you want every activity label replaced with "StoryMaker 2 BETA"?